cypress/events/index.js

```diff
@@ -72,7 +72,11 @@ module.exports = function setupNodeEvents (on, config) {
     config.env.packageFolder = path.join(config.env.projectFolder, 'node_modules', 'govuk-prototype-kit')
   }
 
-  const waitUntilAppRestarts = async (timeout) => await waitOn({ delay: 3000, resources: [config.baseUrl], timeout })
+  const waitUntilAppRestarts = (timeout = 20000) => waitOn({
+    delay: 3000,
+    resources: [config.baseUrl],
+    timeout
+  }).then(() => sleep(2000))
```

Comment:
Do we need to add the forced two second sleep here?  That's going to significantly slow everything down.

Reply:
I'll remove to see what happens.  I made that change earlier in the piece.  It needed it then, maybe not now.

Reply:
Removed

Reply:
I can see the tests failed.  They ran fine on my machine without this additional delay, I'm wondering if we can try two things:

1. A shorter delay
2. Only adding the delay in the CI environment

Reply:
I restarted them and they passed

Reply:
They ran ok on my machine also, sometimes around the middle of the day and the afternoon the tests don't always pass due to the test instances running a little slower I suspect.